src/components/DisplayBalance/DisplayBalance.js

```diff
@@ -0,0 +1,104 @@
+import { ReactComponent as GrowSymbol } from '../../assets/growSymbol.svg'
+import { ReactComponent as ShareSymbol } from '../../assets/shareSymbol.svg'
+import { ReactComponent as SpendSymbol } from '../../assets/spendSymbol.svg'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+DisplayBalance.propTypes = {
+  type: PropTypes.string.isRequired,
+  value: PropTypes.number,
+}
+
+export default function DisplayBalance({ type, value }) {
+  return (
+    <Wrapper>
+      <CircleValueWrapper>
+        <Circle type={type} data-testid={`svg${type}`} color={getColor(type)}>
+          {getSymbol(type)}
+        </Circle>
+        <Value>{value}</Value>
+      </CircleValueWrapper>
+
+      <Title>{type}</Title>
+    </Wrapper>
+  )
+
+  function getColor(type) {
+    const color = {
+      Grow: function () {
+        return 'var(--main-yellow)'
+      },
+      Share: function () {
+        return 'var(--main-green)'
+      },
+      Spend: function () {
+        return 'var(--main-blue)'
+      },
+      default: function () {
+        return ''
+      },
+    }
+    return (color[type] || color['default'])()
+  }
+
+  function getSymbol(type) {
+    const symbol = {
+      Grow: function () {
+        return <GrowSymbol />
+      },
+      Share: function () {
+        return <ShareSymbol />
+      },
+      Spend: function () {
+        return <SpendSymbol />
+      },
+      default: function () {
+        return ''
+      },
+    }
+    return (symbol[type] || symbol['default'])()
+  }
+}
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 140px;
+`
+
+const CircleValueWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`
+const Circle = styled.div`
+  background-color: rgb(${({ color }) => color});
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 60px;
+  height: 60px;
+  border-radius: 50px;
+  margin: 15px;
+`
+
+const Value = styled.span`
+  width: 75px;
+  height: 45px;
+  z-index: -1;
+  background-color: var(--color-babyblue);
+  font-size: 1.3em;
+  font-weight: bold;
+  padding: 15px;
+  border-radius: 15px;
+  text-align: center;
+  margin-left: -40px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`
+
+const Title = styled.span`
```

Comment:
Why do you use a `span` as  a title component? We have the headlines h1 ~ h6, why not use them instead?

Reply:
I have a big problem to decide when to use which headline tag. For the component itself h1 is obviously right. But in the context of the hole project a h1 could be wrong, when we use the component in another context later in the project. A span hurts nobody and give me opportunity to style it. What would be your suggestion for what headline tag we could use?

Reply:
I understand your struggles, I guess here it would make sense to have it as an `h2`, however it is not really clear with all the other styling we have. It just bugged me a bit, that it is called `Title` and then it is a `span`. Maybe `accountName` or something like that?

Reply:
Semantically speaking each section can contain its own hierachy of headings, so a `h1` would be fine I think, but we have to adjust the styling.

Reply:
Hmmmm... I think we only want one `h1` per HTML document and then structure it hierarchically.

Reply:
Interesting.. will do some reading this evening, maybe I got it all wrong 🙈

Reply:
This "one h1 per section" was an idea for html5. Unfortunately it was never implemented by browsers and is now considered outdated. 

You could make the element type dynamic for this component like this: 
`<Title tag="h2">Hello</Title>` and default it to be a span. 

You could add an aria role to make it a headline even though it is a span.